packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -35,46 +39,42 @@ class BindingHitTestEntry extends HitTestEntry {
   final HitTestResult result;
 }
 
-/// State used in converting ui.Event to InputEvent
+/// State used in converting PointerPackets to PointerInputEvents
 class _PointerState {
   _PointerState({ this.pointer, this.lastPosition });
   int pointer;
   Point lastPosition;
 }
 
-class _UiEventConverter {
-  static InputEvent convert(ui.Event event) {
-    if (event is ui.PointerEvent)
-      return convertPointerEvent(event);
-
-    // Default event
-    return new InputEvent(
-      type: event.type,
-      timeStamp: event.timeStamp
-    );
-  }
-
+class _PointerEventConverter {
   // Map actual input pointer value to a unique value
   // Since events are serialized we can just use a counter
   static Map<int, _PointerState> _stateForPointer = new Map<int, _PointerState>();
   static int _pointerCount = 0;
 
-  static PointerInputEvent convertPointerEvent(ui.PointerEvent event) {
-    Point position = new Point(event.x, event.y);
+  static List<PointerInputEvent> convertPointerPacket(PointerPacket packet) {
+    return packet.pointers.map(_convertPointer).toList();
+  }
 
-    _PointerState state = _stateForPointer[event.pointer];
+  static PointerInputEvent _convertPointer(Pointer ptr) {
```

Comment:
done.  Although imho it's a bit confusing to have an object called "pointer" with a field called "pointer" (the pointer index)